.github/workflows/install_script.yml

```diff
@@ -18,11 +18,12 @@ on:
 
 jobs:
   run_install_script:
-    runs-on: ubuntu-latest
     strategy: 
       matrix:
         branch: ['develop', 'stable'] # 'develop' can be anything that's not 'stable'. This triggers an if statement in ubuntu.sh
         build_hdf5: ['NO', 'hdf5-1_12_0']
+        os: ['ubuntu-22.04', 'ubuntu-20.04']
```

Comment:
Now that I accepted @ahnaf-tahmid-chowdhury's change, I realize that our standard testing all happens on `ubuntu-latest` so this was mostly about adding `ubuntu-20.04`.  Thus there may be a larger question about whether we want to use `ubuntu-latest` in our regular testing?

Reply:
For now, the latest version of Ubuntu is 22.04. So, I believe we can re-add ubuntu-latest when 22.04 is released. Alternatively, we can always use latest and create a PR to add 22.04 when Ubuntu 24.04 is released.

Let me know which option we may proceed.

Reply:
Probably makes sense to change all of them to `22.04` so that things don't suddenly break when 24.04 is released.